packages/source-iottwinmaker/src/initialize.ts

```diff
@@ -153,6 +154,7 @@ export const initialize = (
     s3SceneLoader: (sceneId: string) => new S3SceneLoader({ workspaceId, sceneId, twinMakerClient, s3Client }),
     sceneMetadataModule: (sceneId: string) =>
       new SceneMetadataModule({ workspaceId, sceneId, twinMakerClient, secretsManagerClient }),
+    kGDatamodule: (queryStatement: string) => new KGDataModule({ workspaceId, twinMakerClient, queryStatement }),
```

Comment:
should the `queryStatement` be a parameter of the executeQuery function so when the user want to execute multiple different queries, they will do 
```
const kg = initialize(...).kGDatamodule();
...
kg.executeQuery('SELECT A....')
kg.executeQuery('SELECT B....')
kg.executeQuery('SELECT C....')
```
instead of 
```
const ds = initialize(...)
...
ds.kGDatamodule('SELECT A....').executeQuery()
ds.kGDatamodule('SELECT B....').executeQuery()
ds.kGDatamodule('SELECT C....').executeQuery()
```

Reply:
I was debating the same thing, I think you are right your suggestion is better so we can we have one instance of the class. Will fix and re-commit.